userguide/install.rst

```diff
@@ -33,19 +33,90 @@ Getting %brand%
+
+Checking Installer Integrity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+%brand% uses the
+`OpenPGP standard <https://en.wikipedia.org/wiki/Pretty_Good_Privacy#OpenPGP>`__
+to confirm that downloaded files have been provided by a trustworthy
+source. OpenPGP compliant software like
+`gnupg <https://www.freebsd.org/cgi/man.cgi?query=gpg>`__,
+`Kleopatra <https://www.openpgp.org/software/kleopatra/>`__,
+or `Gpg4win <https://gpg4win.org/>`__. can check the PGP signature of a
+%brand% installer file.
+
+The :file:`sha256.txt` file is used to confirm the integrity of the
+downloaded :file:`.iso`. See :ref:`SHA256 Verification` for more
+details.
+
+
+PGP Verification
+^^^^^^^^^^^^^^^^
+
+To verify the :file:`.iso` source, go to
+`<https://www.freenas.org/download-freenas-release/>`__ and click
+:guilabel:`PGP Signature` to download the software signature file. Open
+the :guilabel:`PGP Public key` link and note the browser address and
+:literal:`Search results` string.
+
+Use one of the OpenPGP encryption tools that were mentioned above to
+import the public key and verify the PGP signature.
+
+This example shows verifying the %brand% :file:`.iso` using
+:command:`gpg` in a command prompt:
+
+* Go to the :file:`.iso` and :file:`.iso.gpg` download location and
+  import the public key using the keyserver address and search results
+  string:
+
+.. code-block:: none
+
+   tmoore@Observer ~> cd Downloads/
+   tmoore@Observer ~/Downloads> gpg --keyserver sks-keyservers.net --recv-keys 0xc8d62def767c1db0dff4e6ec358eaa9112cf7946
+   gpg: /usr/home/tmoore/.gnupg/trustdb.gpg: trustdb created
+   gpg: key 358EAA9112CF7946: public key "IX SecTeam <security-officer@ixsystems.com>" imported
+   gpg: Total number processed: 1
+   gpg:               imported: 1
+   tmoore@Observer ~/Downloads>
+
+* Use :command:`gpg --verify` to compare the :file:`.iso` and
+  :file:`.iso.gpg` files:
+
+.. code-block:: none
+
+   tmoore@Observer ~/Downloads> gpg --verify FreeNAS-11.2-U6.iso.gpg FreeNAS-11.2-U6.iso
+   gpg: Signature made Tue Nov  5 13:48:18 2019 EST
+   gpg:                using RSA key C8D62DEF767C1DB0DFF4E6EC358EAA9112CF7946
+   gpg: Good signature from "IX SecTeam <security-officer@ixsystems.com>" [unknown]
+   gpg: WARNING: This key is not certified with a trusted signature!
+   gpg:          There is no indication that the signature belongs to the owner.
+   Primary key fingerprint: C8D6 2DEF 767C 1DB0 DFF4  E6EC 358E AA91 12CF 7946
+   tmoore@Observer ~/Downloads>
+
+* This response means the signature is correct but still untrusted. Go
+  back to the browser page that has the :guilabel:`PGP Public key` open
+  and manually confirm that the key was issued for the iX Security Team
+  on October 15, 2019 and has been signed by iXsystems accounts.
+
+.. _SHA256 Verification:
+
+SHA256 Verification
+^^^^^^^^^^^^^^^^^^^
+
+The command to verify the checksum varies by operating system:
 
 * on a BSD system use the command
   :samp:`sha256 {name_of_file}`
```

Comment:
This can be more helpful to the reader by telling them which file:
```suggestion
  :samp:`sha256 {isofile}`
```

Reply:
This is technically out of scope for this PR/ticket.